RSP.java

```diff
@@ -0,0 +1,112 @@
+import java.util.Scanner;
```

Comment:
Add this file to a folder.
folder name should be Rock_Paper_Scissor
And file name should be Rock_Paper_Scissor.java